docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -0,0 +1,285 @@
+
+5.  Within the `Initialize()` method, set the text label properties:
+
+    1.  Create the text label object from the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+        ```
+        TextLabel text = new TextLabel("Hello Tizen NUI World");
+        ```
+
+    2.  Align the text horizontally to the center of the available area:
+        ```
+        text.HorizontalAlignment = HorizontalAlignment.Center;
+        ```
+
+    3.  To illustrate the label width, set the label text color:
+        ```
+        text.TextColor = Color.Blue;
+        ```
+
+    4.  Define the text size in points:
+        ```
+        text.PointSize = 12.0f;
+        ```
+
+    For more information on the key properties of the `Tizen.NUI.BaseComponents.TextLabel` class, see [TextLabel](../../guides/nui/textlabel.md).
+
+6.  Implement the main application window:
+    1.  Create the window and add an event handler for the `KeyEvent` event of the [Tizen.NUI.Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Window.html) class. This event handler is invoked whenever the key event is received.
+
+        ```
+        Window.Instance.KeyEvent += OnKeyEvent;
+        ```
+
+    2.  Add the text label to the window's root layer:
+
+        ```
+        Window.Instance.GetDefaultLayer().Add(text);
+        ```
+
+7.  Define the event handler to terminate the NUI application:
+
+    ```
+    public void OnKeyEvent(object sender, Window.KeyEventArgs e)
+    {
+        if (e.Key.State == Key.StateType.Down && (e.Key.KeyPressedName == "XF86Back" || e.Key.KeyPressedName == "Escape"))
+        {
+            Exit();
+        }
+    }
+    ```
+
+8.  To handle behavior when the window close button is clicked and the application is about to terminate, override the `OnTerminate()` method of the `Tizen.NUI.NUIApplication` class:
+
+    ```
+    protected override void OnTerminate()
+    {
+        base.OnTerminate();
+    }
+    ```
+
+    > **Note**
+    >
+    > To invoke the `Terminated` event of the `Tizen.Applications.CoreApplication` class, you must call the `base.OnTerminate()` method inside the override.
+
+9.  Implement the `Main()` method:
+    1.  Create the application through the default constructor:
+
+        ```
+        static void Main(string[] args)
+        {   
+            var app = new Program();
+            app.Run(args);
+        }
+        ```
+
+    2.  Start the application main loop.
+
+        This ensures that images are displayed, and that events and signals are dispatched and captured.
+
+        In this simple tutorial, the `Main()` method resides within the class. For significant application development, the `Main()` method must be placed in a separate `.cs` file.
+
+        ```
+        app.Run(args);
+        ```
+
+## Build and Run NUI Application
+1.  Building the solution:
```

Comment:
To Build and Run the NUI application, follow these steps: 

1.  Build the solution: